myapp/handlers/spotify-handler.go

```diff
@@ -186,6 +186,34 @@ func (h *Handlers) SetSpotifySongsForUser(userID int, songs spotify.FullTrackPag
 		return err
 	}
 
+	fmt.Println("STARTING GETTING SONGS' ANALYSES FROM SPOTIFY.")
+	songID1 := songs.Tracks[0].ID
+	songID2 := songs.Tracks[1].ID
+	songID3 := songs.Tracks[2].ID
+	songID4 := songs.Tracks[3].ID
+	songID5 := songs.Tracks[4].ID
+	songID6 := songs.Tracks[5].ID
+	songID7 := songs.Tracks[6].ID
+	songID8 := songs.Tracks[7].ID
+	songID9 := songs.Tracks[8].ID
+	songID10 := songs.Tracks[9].ID
+	songID11 := songs.Tracks[10].ID
+	songID12 := songs.Tracks[11].ID
+	songID13 := songs.Tracks[12].ID
+	songID14 := songs.Tracks[13].ID
+	songID15 := songs.Tracks[14].ID
+	songID16 := songs.Tracks[15].ID
+	songID17 := songs.Tracks[16].ID
+	songID18 := songs.Tracks[17].ID
+	songID19 := songs.Tracks[18].ID
+	songID20 := songs.Tracks[19].ID
```

Comment:
This needs to be an array that you iterate over, this causes my instance to panic. My account only has a top 8 songs.

Reply:
This is still causing a panic